src/test/scala/chiselTests/ChiselSpec.scala

```diff
@@ -154,22 +154,23 @@ trait ChiselRunners extends Assertions {
       val testPoint = gen
       assert(!testPoint.isWidthKnown, s"Asserting that width should be inferred yet width is known to Chisel!")
       // Sanity check that firrtl doesn't change the width
-      val out = IO(Output(chiselTypeOf(testPoint)))
+      val widthcheck = Wire(Output(chiselTypeOf(testPoint)))
```

Comment:
Does this need `Output` here to strip flips?

Reply:
That makes sense, and matches how it worked before.  Will `:=` hook everything up from `testPoint` if the dest is coerced / Output?
(Basically wondering if this presently supports things with flips or not)

Anyway good catch ty I didn't mean to change this, will fix :+1: .

Reply:
Whoops, I didn't change this, you're asking if it needs this ("still")?

Inclined to leave it as-is since it was present before but I'm not sure the story here re:expectations/assumptions/behavior.

Tests pass without it but that doesn't mean it's right to drop but just FYI.

Reply:
These tests are assuming passive types so I don't think it matters either way, I think it's fine to keep or remove.

Reply:
Dropped here and from sibling checker.